main.go

```diff
@@ -110,7 +110,14 @@ func main() {
 		ginutil.RecoverProblem,
 	)
 
-	if config.HTTP.CORS.AllowAllOrigins {
+	if len(config.HTTP.CORS.AllowOrigins) > 0 {
+		log.Info().Message("Allowing " + config.HTTP.CORS.AllowOrigins + " origins in CORS.")
```

Comment:
nit: A rule of thumb I've used for logs is to not have anything varying in the message.

Suggest moving the config value here to a `WithString` call. For example:

```suggestion
		log.Info().WithString("origin", config.HTTP.CORS.AllowOrigins).
			Message("Allowing origins in CORS.")
```

But if you're changing to a string slice, then perhaps just something like this:

```suggestion
		log.Info().WithStringf("origin", "%v", config.HTTP.CORS.AllowOrigins).
			Message("Allowing origins in CORS.")
```

^ would resolve to something like ``origins=`[http://localhost:5001 http://localhost:4200]` `` in the console output for multiple values

Otherwise a more custom solution:


```suggestion
		log.Info().WithString("origin", strings.Join(config.HTTP.CORS.AllowOrigins, " ")).
			Message("Allowing origins in CORS.")
```

^ would resolve to something like ``origins=`http://localhost:5001 http://localhost:4200` `` in the console output for multiple values

Reply:
Sol 2 seems to work